spacepy/pybats/bats.py

```diff
@@ -863,18 +889,90 @@ def calc_j(self):
 
         self['j'] = sqrt(self['jx']**2.0 + self['jy']**2.0 + self['jz']**2.0)
         self['j'].attrs = {'units':self['jx'].attrs['units']}
-    
+
+    @calc_wrapper
+    def calc_uperp(self):
+        '''
+        Calculate the magnitude of the velocity perpendicular to the
+        magnetic field: $\vec{U} \times \hat{b}$.  Result maintains units
+        of velocity.
+
+        Values are calculated for each fluid and stored as self['u_perp']
+        (using species prefixes as needed).
+
+        Notes
+        =====
+        .. versionadded:: 0.5.0
+        '''
+
+        from numpy import sqrt
```

Comment:
numpy is imported as `np` at the top of the file, why are we doing another import here instead of using `np.sqrt`?